src/main/java/com/mongodb/kafka/connect/util/ConfigHelper.java

```diff
@@ -48,7 +53,13 @@ private ConfigHelper() {}
             Document.parse(format("{s: %s}", jsonArray)).getList("s", Document.class);
```

Comment:
This is the cause of escaping problems - Using format on the string means that users had to set extra escaping in their configuration.